tests/integration/cattletest/core/test_account.py

```diff
@@ -3,7 +3,7 @@
 
 
 def test_account_create(admin_client):
-    cred_count = len(admin_client.list_credential())
+    cred_count = len(admin_client.list_credential(limit=1000))
```

Comment:
I think the assertion is good, it will help some poor developer troubleshoot this.